source/AI.cpp

```diff
@@ -3004,6 +3003,50 @@ bool AI::DoCloak(Ship &ship, Command &command)
 
 
 
+void AI::DoPatrol(Ship &ship, Command &command) const
+{
+	double radius = ship.GetSystem()->ExtraHyperArrivalDistance();
```

Comment:
How is this going to work if the system has no set arrival distance?

Reply:
I guess I could add a minimum of 1000 or something?